lib/ansible/config/base.yml

```diff
@@ -328,6 +328,16 @@ COLOR_WARN:
   env: [{name: ANSIBLE_COLOR_WARN}]
   ini:
   - {key: warn, section: colors}
+COMMON_REMOTE_GROUP:
```

Comment:
don't do this in base, do it in shell plugin so you can also allow per host vars

Reply:
missing version_added

Reply:
Added `version_added`. Curious what others (@sivel, @nitzmahone, @abadger) think about where this should be.

I actually did originally have this in the shell plugin config, but moved it out to base for two reasons:

* This is a setting needed with every shell; it shouldn't be up to a shell plugin to decide if this gets disabled. Changing to a group is a platform abstraction, not a shell abstraction, and platform abstractions are currently managed by the connection plugin (see `_IS_WINDOWS`), not the shell.
* This only belongs with and affects `fixup_perms2()` which is in action, not shell, and therefore a different subsystem.

Reply:
not all platforms work the same, some like windows always have acls, others like networking os make all of this irrelevant, others even allow direct assignment to user w/o having to resort to groups.

Reply:
Ok, moving it to shell.